docs/src/docs/extending-testerra/hooks.adoc

```diff
@@ -16,7 +16,7 @@ public class ConfigureMyModule extends AbstractModule {
 }
 ----
 
-IMPORTANT: Please be aware, that the package namespace prefix `eu.tsystems.mms.tic` is required in order to find your module configuration.
+IMPORTANT: Please be aware, that the package namespace prefix `io.testerra` is required in order to find your module configuration.
```

Comment:
Not really, because
1) I cannot link from docs to a class member.
2) I want to hide the outdated package `eu.tsystems.mms.tic`. `io.testerra` is the new standard (#8)

Reply:
Fair enough on #2.

([Relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) ought to work.)